### PR TITLE
Related items outfit creation wire frame

### PR DESCRIPTION
**Pull request by joshgarza**
Created OutfitCard Component
Added onClick in OutfitCreation component to add new Outfit

### CODE REVIEW

File: client/src/components/RelatedItems/OutfitCard.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        let reviews = result.data.results;
+        calcRating(reviews);
+      });
+
+    axios.get(`/products/${product.id}/styles`)
+      .then(result => {
+        let styles = result.data.results;
+
+        for (let i = 0; i < styles.length; i++) {
+          if (styles[i]['default?'] === true) {
```
Comment (stephaaniechen) [inline client/src/components/RelatedItems/OutfitCard.jsx:64]: what if there's a sale price? do the cards only show the original price?

File: client/src/components/RelatedItems/OutfitCard.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            setPrice(currStyle => {
+              return styles[i].original_price;
```
Comment (stephaaniechen) [inline client/src/components/RelatedItems/OutfitCard.jsx]: what if there's a sale price? do the cards only show the original price?

File: client/src/components/RelatedItems/OutfitCard.jsx (hunk 1 above)
Comment (joshgarza) [reply client/src/components/RelatedItems/OutfitCard.jsx:64]: I totally did not account for that yet. Right now they only show the original price or the default price if no default was ever set. Lemme make that change real quick

**Review by stephaaniechen — APPROVED** [review]
👌🏻

**Pull request merged**